### DISCUSSION TITLE
Zoom Feature Enhancement for Scatter Plots

### DISCUSSION THREAD

**Discussion by unidesigner** [Kanaries/graphic-walker#55] [Ideas]
Would it be possible to add a way to zoom into a scatter plot by drawing a rectangle with the mouse for the zoom area? It would be quite useful, e.g. see the following example plot

![Screenshot from 2023-04-12 11-24-55](https://user-images.githubusercontent.com/136799/231416334-8774fee1-9291-41dc-ade9-e0732e253ba2.png)

**Comment by jojocys** [comment]
Yes, you can enable the axis resize feature to zoom in/out
<img width="489" alt="image" src="https://github.com/Kanaries/graphic-walker/assets/59559181/3f2abc94-58a6-47b6-b5eb-23a7aaaa571d">
